README.md

```diff
@@ -16,17 +16,25 @@ crate.
+| -------------------- | --------------------- | --------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ | ---- | -------- |
+| [`aes-gcm-siv`]      | [AES-GCM-SIV]         | [![crates.io](https://img.shields.io/crates/v/aes-gcm-siv.svg)](https://crates.io/crates/aes-gcm-siv)           | [![Documentation](https://docs.rs/aes-gcm-siv/badge.svg)](https://docs.rs/aes-gcm-siv)           | 1.51 | 💚       |
+| [`aes-gcm`]          | [AES-GCM]             | [![crates.io](https://img.shields.io/crates/v/aes-gcm.svg)](https://crates.io/crates/aes-gcm)                   | [![Documentation](https://docs.rs/aes-gcm/badge.svg)](https://docs.rs/aes-gcm)                   | 1.51 | 💚       |
+| [`aes-siv`]          | [AES-SIV]             | [![crates.io](https://img.shields.io/crates/v/aes-siv.svg)](https://crates.io/crates/aes-siv)                   | [![Documentation](https://docs.rs/aes-siv/badge.svg)](https://docs.rs/aes-siv)                   | 1.51 | 💚       |
+| [`ascon-aead`]       | [Ascon]               | [![crates.io](https://img.shields.io/crates/v/ascon-aead.svg)](https://crates.io/crates/ascon-aead)             | [![Documentation](https://docs.rs/ascon-aead/badge.svg)](https://docs.rs/ascon-aead)             | 1.56 | 💚       |
+| [`ccm`]              | [CCM]                 | [![crates.io](https://img.shields.io/crates/v/ccm.svg)](https://crates.io/crates/ccm)                           | [![Documentation](https://docs.rs/ccm/badge.svg)](https://docs.rs/ccm)                           | 1.41 | 🤍       |
+| [`chacha20poly1305`] | [(X)ChaCha20Poly1305] | [![crates.io](https://img.shields.io/crates/v/chacha20poly1305.svg)](https://crates.io/crates/chacha20poly1305) | [![Documentation](https://docs.rs/chacha20poly1305/badge.svg)](https://docs.rs/chacha20poly1305) | 1.56 | 💚       |
+| [`deoxys`]           | [Deoxys-I/II]         | [![crates.io](https://img.shields.io/crates/v/deoxys.svg)](https://crates.io/crates/deoxys)                     | [![Documentation](https://docs.rs/deoxys/badge.svg)](https://docs.rs/deoxys)                     | 1.51 | 💚       |
+| [`eax`]              | [EAX]                 | [![crates.io](https://img.shields.io/crates/v/eax.svg)](https://crates.io/crates/eax)                           | [![Documentation](https://docs.rs/eax/badge.svg)](https://docs.rs/eax)                           | 1.41 | 💚       |
+| [`mgm`]              | [MGM]                 | [![crates.io](https://img.shields.io/crates/v/mgm.svg)](https://crates.io/crates/mgm)                           | [![Documentation](https://docs.rs/mgm/badge.svg)](https://docs.rs/mgm)                           | 1.41 | 🤍       |
```

Comment:
Why it's marked by 🤍? AFAIK it's the only AEAD algorithm standardized in Russia.